src/components/page-list/page-list.css

```diff
@@ -0,0 +1,33 @@
+/* ===== List View ===== */
+.page-list {
+  table-layout: fixed;
+}
+
+.page-list>tbody>tr {

```

Comment:
Can you add a space between each selector like so? `.page-list > tbody > tr` 

Same for each of the styles below.